Adam9500/integrate with envoy (#7)
racker/salus-oracle-agent: README.md

> Provides an agent that gathers information from exported data from oracle for reporting to a monitoring systems such as Salus.

Reviewer: ```suggestion
Provides an agent that gathers information from exported data from Oracle for reporting to a monitoring system such as Salus.
```

> Provides an agent that gathers information from exported data from oracle for reporting to a monitoring systems such as Salus.

Reviewer: I wonder if it's worth briefly clarifying that it targets "internal support methodologies"...just in case someone Googles and gets excited to find this "Oracle monitoring agent"

## Usage

```
  -configs string (default: ./config.d)
    	directory containing config files that define continuous monitoring (env AGENT_CONFIGS)
```

## Continuous-Monitoring Config File Format

> When running the agent it will periodically collect package telemetry at the interval configured in each config file. The option specifies a directory where any files in that directory that is of the correct format will be processed. The files are expected in JSON format and the structure of those files is:

Reviewer: Your copy-paste is showing ;) 

```suggestion
When running the agent it will periodically collect Oracle telemetry at the interval configured in each config file. The option specifies a directory where any files in that directory that is of the correct format will be processed. The files are expected in JSON format and the structure of those files is:
```

```json
{
  "interval":  30,
  "type":"oracle_rman",
  "databaseNames":["RMAN"],
  "filePath":"./testdata/",
  "errorCodeWhitelist":  ["RMAN-1234"]
}
```

where:

> - `interval` : a Go int specifying the interval of package collection, in seconds.

Reviewer: Could just say
```suggestion
- `interval` : an integer specifying the interval of package collection, in seconds.
```

- `type` : indicates the type or oracle monitoring you would like to do. Values must be one of the following "oracle_rman", "oracle_tablespace", or "oracle_dataguard"
- `filePath` : This filePath is the location of where we expect to find the files associated with the database names. 

> - `databaseNames` : This is a list of the names of the databases that we want to monitor. These are expected to be .txt files of the same name as in this list in the filePath provided. 

Reviewer: Your mixture of "we" and "you" throughout this section is a bit inconsistent. As you know, I recommend avoiding those words in technical docs in general.

Author: Doh.. Yeah still working on keeping all of that straight.

```

where:
- `interval` : an integer specifying the interval of package collection, in seconds.
- `type` : indicates the type or oracle monitoring to be started. Values must be one of the following "oracle_rman", "oracle_tablespace", or "oracle_dataguard"

> - `filePath` : This filePath is the location of where it is expect to find the files associated with the database names. 

Reviewer: ```suggestion
- `filePath` : This filePath is the location where the agent expects to find the files associated with the database names. 
```
